Dockerfile

```diff
@@ -12,6 +13,7 @@ ENV NODE_ENV=production
 EXPOSE 3000
 COPY yarn.lock package.json ./
 RUN yarn
```

Comment:
you could remove line 9 in favour of this here 
``` Dockerfile
RUN apk --no-cache add python build-base && \
    yarn &&\
    npm rebuild bcrypt --build-from-source && \
    apk del build-base python && \
    rm -rf /tmp/* /root/.npm /root/.node-gyp
```
this would produce a smaller and clearner container